ghost/ghost/src/index.ts

```diff
@@ -10,6 +11,11 @@ export async function create() {
     return app;
 }
 
+export function addProvider(obj: ClassProvider | ValueProvider) {
+    AppModule.providers?.push(obj);
+    AppModule.exports?.push(obj.provide);
```

Comment:
Exposing as part of exports is only here to be able to configure the dependency incjection, right? As in we should not be referencing the app module directly from anywhere in the http code? 

If that's the case maybe exposing a more specific separate variable like `InjectableValues` or something more descriptive would be better 🤔  

I'm mostly guessing here and trying to understand why are we pushing straight to the `exports`

Reply:
This is not the "node" concept of `module.exports` - this is the Nest concept of `exports` which is a list of providers which are able to be consumed from outside of the module